libsolidity/interface/StandardCompiler.cpp

```diff
@@ -605,7 +605,27 @@ std::variant<OptimiserSettings, Json::Value> parseOptimizerSettings(Json::Value
 		if (details.isMember("yulDetails"))
 		{
 			if (!settings.runYulOptimiser)
+			{
+				auto const& yulDetails = details["yulDetails"];
+				if (yulDetails.isObject() && yulDetails.isMember("optimizerSteps"))
+				{
+					auto error = checkOptimizerDetailSteps(details["yulDetails"], "optimizerSteps", settings.yulOptimiserSteps, settings.yulOptimiserCleanupSteps);
+					auto isWhiteSpaceOrNewlineOnly = [](std::string const& _sequence) {
+						return !std::any_of(_sequence.begin(), _sequence.end(), [](char const step){
+							return step != ' ' && step != '\n';
+						});
+					};
+					if (
+						!error &&
+						isWhiteSpaceOrNewlineOnly(settings.yulOptimiserSteps) &&
+						isWhiteSpaceOrNewlineOnly(settings.yulOptimiserCleanupSteps)
+					)
+						return { std::move(settings) };
```

Comment:
This is not a code block:

```suggestion
						return {std::move(settings)};
```

Reply:
Copied the style from the bottom of the function; also, if `check_style.sh` doesn't complain, is the style wrong? :)